tests/installation/welcome.pm

```diff
@@ -149,6 +150,14 @@ sub run {
         }
         assert_screen('select-product-' . $product);
     }
+    # Accept the License on installations where License Agreement is shown on Welcome screen.
+    elsif (has_license_on_welcome_screen) {
+        if (get_var('INSTALLER_EXTENDED_TEST')) {
+            verify_license_has_to_be_accepted;
+            verify_license_translations unless is_sle('15+');
+        }
+        accept_license;
+    }
```

Comment:
Please correct me if I'm wrong, but it seems like the changes also came from the merge and should not be presented in your PR.

Reply:
very likely. i will try to fix it